package.json

```diff
@@ -8,9 +8,8 @@
     "repository": "duckduckgo/duckduckgo-privacy-extension",
     "scripts": {
         "install-ci": "npm ci --ignore-scripts && npm rebuild puppeteer && npm run postinstall",
-        "postinstall": "git submodule update --init --recursive && npm run copy-sjcl && export PATH='$PATH:$PWD/node_modules/chromedriver/bin'",
+        "postinstall": "git submodule update --init --recursive && cd shared/content-scope-scripts && npm install && export PATH='$PATH:$PWD/node_modules/chromedriver/bin'",
```

Comment:
Depending on whether building the submodule should be part of the usual flow I'd consider skipping the `npm install` here.